src/main.ts

```diff
@@ -14,6 +14,9 @@ import { autoUpdate } from './lib/updater';
 const WindowIcon = nativeImage.createFromPath(path.join(__dirname, "icon.png"));
 WindowIcon.setTemplateImage(true);
 
+let tray;
+const contextMenu = Menu.buildFromTemplate([{icon: WindowIcon, label: 'Revolt', enabled: false}, /* revolt doesn't have credits page yet {label: 'Credits' click: require("shell").openExternal("https://revolt.chat")}*/ { label:'Quit Revolt', click() { app.quit() } }]);
```

Comment:
```suggestion
const contextMenu = Menu.buildFromTemplate([{ icon: WindowIcon, label: `Revolt : Version: ${version}`, enabled: false }, {  label: 'Separator', type: 'separator' }, { label: 'Contribute', click() { shell.openExternal("https://github.com/revoltchat") } }, /* revolt doesn't have credits page yet { label: 'Credits', click() { shell.openExternal("https://revolt.chat") } },*/ {  label: 'Separator', type: 'separator' }, { label:'Quit Revolt', click() { app.quit() } }]);
```
Added: Separators, Contribute, Version, Fixes and clean up (Kinda).

Reply:
This should make the context menu look cleaner and added some fixes to your shell requirement. 

It was inside the import from the electron package and all you really needed to do was `shell.openExpernal`  not `require("shell").openExternal`

Hope that helps you with your future context menus you work on! 😊

Reply:
> 
> 
> This should make the context menu look cleaner and added some fixes to your shell requirement.
> 
> It was inside the import from the electron package and all you really needed to do was `shell.openExpernal` not `require("shell").openExternal`

Thanks for the reply! I've also realized that just now.